tasks/extract-binaries-from-image/README.md

```diff
@@ -5,13 +5,20 @@ Tekton task that extracts binaries to be released on github.com from an image.
 The path to the directory inside the provided workspace where the binaries were
 saved is provided as a result.
 
+The binaries must be stored at the same `image_binaries_path` for each component
```

Comment:
Can you reword this? I don't think I follow

Reply:
We don't need to expand the mapping, just reword this. Can you just remove `when multiple components with binaries area to be extracted` in favor of like `passed`? So it would say `The binaries must be stored at the same `image_binaries_path` for each component passed`. That is clearer to me at least (I think reading binaries so many times in one sentence has me lost).

Reply:
Reworded. Thanks!